.buildkite/pull-requests.org-wide.json

```diff
@@ -11,13 +11,16 @@
       ],
       "build_on_commit": true,
       "build_on_comment": true,
-      "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))",
-      "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))",
+      "trigger_comment_regex": "run docs-build ?(?<rebuild_opt>rebuild)? ?(?<warn_opt>warnlinkcheck)? ?(?<skip_opt>skiplinkcheck)?",
+      "always_trigger_comment_regex": "run docs-build ?(?<rebuild_opt>rebuild)? ?(?<warn_opt>warnlinkcheck)? ?(?<skip_opt>skiplinkcheck)?",
```

Comment:
I haven't tested these regular expressions but I'll assume they work. 

@bmorelli25 @jrodewig It's probably good to note for when the documentation is updated that the options need to be in order. I don't know how the new Buildkite script handles them; in Jenkins it just searched for the "rebuild"/"skiplinkcheck" text anywhere in the comment body. But in terms of a trigger phrase I think they'll need to be in this order.